crates/frontend/src/pages/home.rs

```diff
@@ -303,8 +303,10 @@ pub fn home() -> impl IntoView {
         }
         // resolve the context and get the config that would apply
         spawn_local(async move {
-            let context = gen_query_context(context_rs.get());
-            let mut config = match resolve_config(tenant_rs.get(), context).await.unwrap()
+            let context = gen_query_context(context_rs.get_untracked());
```

Comment:
Why replace with `get_untracked` here? If `get` is not reactive here then move the `get` call to outside of async block. Let's keep the `get`.